chat-widget/src/components/notificationpanestateful/NotificationPaneStateful.tsx

```diff
@@ -0,0 +1,112 @@
+            notificationIconClassName: notificationProps.styleProps?.classNames?.notificationIconClassName ?? defaultChatDisconnectStyleProps.classNames?.notificationIconClassName,
+            notificationIconContainerStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.notificationIconContainerStyleProps, notificationProps.styleProps?.notificationIconContainerStyleProps),
+            hideDismissButton: notificationProps.controlProps?.hideDismissButton ?? defaultChatDisconnectControlProps.hideDismissButton,
+            dismissButtonProps: Object.assign({ onClick: handleDismissNotification }, defaultChatDisconnectControlProps.dismissButtonProps, notificationProps.controlProps?.dismissButtonProps),
+            dismissButtonStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.dismissButtonStyleProps, notificationProps.styleProps?.dismissButtonStyleProps),
+            dismissButtonHoverStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.dismissButtonHoverStyleProps, notificationProps.styleProps?.dismissButtonHoverStyleProps),
+            dismissButtonClassName: notificationProps.styleProps?.classNames?.dismissButtonClassName ?? defaultChatDisconnectStyleProps.classNames?.dismissButtonClassName,
+            hideCloseChatButton: notificationProps.controlProps?.hideCloseChatButton ?? defaultChatDisconnectControlProps.hideCloseChatButton,
+            closeChatButtonProps: Object.assign({ onClick: onCloseChatClick }, defaultChatDisconnectControlProps.closeChatButtonProps, notificationProps.controlProps?.closeChatButtonProps),
+            closeChatButtonStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.closeChatButtonStyleProps, notificationProps.styleProps?.closeChatButtonStyleProps),
+            closeChatButtonHoverStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.closeChatButtonHoverStyleProps, notificationProps.styleProps?.closeChatButtonHoverStyleProps),
+            closeChatButtonClassName: notificationProps.styleProps?.classNames?.closeChatButtonClassName ?? defaultChatDisconnectStyleProps.classNames?.closeChatButtonClassName,
+            infoGroupStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.infoGroupStyleProps, notificationProps.styleProps?.infoGroupStyleProps),
+            buttonGroupStyleProps: Object.assign({}, defaultChatDisconnectStyleProps.buttonGroupStyleProps, notificationProps.styleProps?.buttonGroupStyleProps),
+        };
+    };
+
+    switch (notificationScenarioType) {
+        case NotificationScenarios.ChatDisconnect:
+            populateInternalProps(notificationPaneProps?.chatDisconnectNotificationProps);
+            break;
+        default:
+            // setting chat disconnect as default case for now until further customization support is added
+            populateInternalProps(notificationPaneProps?.chatDisconnectNotificationProps);
```

Comment:
I think we should just break in case notificationScenarioType is unknown? Otherwise if there is an unexpected error that triggers this, the user sees chat disconnect for no reason

Reply:
Removed default